FogCreek/Glitch-Community
# Apple tang: bring back notes to the homepage

## Links
* https://apple-tang.glitch.me
* https://glitch.manuscript.com/f/cases/3328336/

## GIF/Screenshots:
On the homepage:
![Screen Shot 2019-04-18 at 10 19 26 AM](https://user-images.githubusercontent.com/6620164/56367571-85a1c800-61c3-11e9-83e5-f658d3380bcf.png)
In a collection:
![Screen Shot 2019-04-18 at 10 19 38 AM](https://user-images.githubusercontent.com/6620164/56367573-85a1c800-61c3-11e9-8f5f-afa8c6698e34.png)

## Changes:
* bring back notes to the homepage

## How To Test:
* add `  { owner: 'glitch', name: 'sketch-with-p-5' }` to src/curated/collections.js so you can see a collection with notes in it on the homepage (plz remove when done!)
* verify you can edit notes on collections you own on the collections page but not on the homepage
* verify you can't edit notes on collections you don't own
* verify that ProjectsUL looks good in all the places it's used:
    * homepage collections
    * user and team pages: pinned projects + "recent projects" 
    * project page "related projects"
    * category page
    * collection page
     
## Feedback I'm looking for:
* this is right yeah? Just want to make sure we're not losing any style fixes as well. One thing that seems noticable to me is the distance between the note and the faces is different on the homepage than on the collection page, but I'm thinking that might be intentional seeing as there's css in the code to make that happen. (UPDATE: yes the spacing was weird, yes it's been fixed!)
